src/main/java/org/jitsi/jigasi/transcription/TranslationManager.java

```diff
@@ -0,0 +1,162 @@
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import org.jitsi.jigasi.*;
+
+import java.util.*;
+
+/**
+ * This class manages the translations to be done by the transcriber.
+ *
+ * @author Praveen Kumar Gupta
+ */
+public class TranslationManager
+    implements TranscriptionListener
+{
+
+    /**
+     * Property name for getting a target language for translation.
+     */
+    public final static String P_NAME_DEFAULT_TARGET_LANGUAGE
+            = "org.jitsi.jigasi.transcription.DEFAULT_TARGET_LANGUAGE";
+
+    /**
+     * Whether to translate text before sending results and the target language.
+     */
+    public final static String DEFAULT_TARGET_LANGUAGE
+            = "";
+
+    /**
+     *
+     */
+    private ArrayList<String> languages  = new ArrayList<>();
+
+    /**
+     * List of listeners to be notified about a new TranslationResult.
+     */
+    private ArrayList<TranslationResultListener> listeners = new ArrayList<>();
+
+    /**
+     * The translationService to be used for translations.
+     */
+    private final TranslationService translationService;
+
+    /**
+     * Initializes the translationManager with a TranslationService
+     * and adds the default target language to the list.
+     *
+     * @param translationService to be used by the TranslationManger
+     */
+    public TranslationManager(TranslationService translationService)
+    {
+        this.translationService = translationService;
+        String defaultTargetLanguage = getTargetLanguage();
+
+        if(!defaultTargetLanguage.isEmpty())
+        {
+            addTargetLanguage(defaultTargetLanguage);
+        }
+    }
+
+    /**
+     * Adds a {@link TranslationResultListener} to the list of
+     * listeners to be notified of a TranslationResult.
+     *
+     * @param listener to be added.
+     */
+    public void addListener(TranslationResultListener listener)
+    {
+        this.listeners.add(listener);
+    }
+
+    /**
+     * Adds a language tag to the list of target languages for
+     * translation.
+     *
+     * @param language to be added to the list
+     */
+    public void addTargetLanguage(String language)
+    {
+        this.languages.add(language);
+    }
+
+    /**
+     * Translates the received {@link TranscriptionResult} into required languages
+     * and returns a list of {@link TranslationResult}s.
+     *
+     * @param result the TranscriptionResult notified to the TranslationManager
+     * @return list of TranslationResults
+     */
+    public ArrayList<TranslationResult> getTranslations(TranscriptionResult result)
+    {
+        ArrayList<TranslationResult> translatedResults
+            = new ArrayList<>();
+        for(String language : languages)
+        {
+            String translatedText = translationService.translate(
+                result.getAlternatives().iterator().next().getTranscription(),
+                result.getLanguage().substring(0,2),
```

Comment:
Translation Service does not need country tags. e.g. "en-US" is not required. "en" is enough.
Temporary solution. Working on managing the languages in a better way.